plugins/get_url/plugin.spec.yaml

```diff
@@ -4,22 +4,23 @@ products: [insightconnect]
 name: get_url
 title: Get URL
 description: Get URL plugin downloads files by URL using HTTP, HTTPS, or FTP
-version: 2.0.0
+version: 2.0.1
+supported_versions: ["_There are no supported product versions listed._"]
```

Comment:
```suggestion
supported_versions: []
```

Reply:
After this change we get error:
```
Validator "SupportedVersionValidator" failed!
        Cause: supported_versions list does not contain values.
```

Reply:
Oh right...you can leave it this way. @mberezin-r7 is there a standard here for first party plugins that don't have a vendor version at all?